OpenSim/Simulation/SimbodyEngine/PinJoint.h

```diff
@@ -42,35 +42,16 @@ are in the desired direction.
 \image html pinJoint.gif
 
  @author Ajay Seth
- @version 1.0
  */
 class OSIMSIMULATION_API PinJoint : public Joint {
 OpenSim_DECLARE_CONCRETE_OBJECT(PinJoint, Joint);
 
-private:
-    static const int _numMobilities = 1;
-//=============================================================================
-// DATA
-//=============================================================================
-protected:
-
-    /** Pin has no additional properties*/
+    /** Specify the Coordinate of the PinJoint */
+    CoordinateP rz{ constructCoordinate(Coordinate::MotionType::Rotational) };
 
-//=============================================================================
-// METHODS
-//=============================================================================
 public:
-    // CONSTRUCTION
-    PinJoint();
-    // Convenience constructor
-    PinJoint(const std::string &name,  const PhysicalFrame& parent,
-             const SimTK::Vec3& locationInParent, const SimTK::Vec3& orientationInParent,
-             const PhysicalFrame& child,
-             const SimTK::Vec3& locationInchild, const SimTK::Vec3& orientationInChild,
-             bool reverse = false);
-    virtual ~PinJoint();
-
-    int numCoordinates() const override {return _numMobilities;};
+    /** Use Joint's constructors. @see Joint */
```

Comment:
Does this look okay if you look at the generated Doxygen html? I think usually the `@see` goes on its own line.